manifests/init.pp

```diff
@@ -52,6 +52,8 @@
 #   $statsd_forward_port
 #       Set the value of the statsd_forward_port varable. Used to forward all
 #       statsd metrics to another host.
+#   $check_freq
+#       Sets how often checks should be run. By default, set to 15s
```

Comment:
This will break us if we dont have that. If its exposed by the dd-agent and documented why cant it be set by the module?